website/docs/providers/proxy/_nginx_ingress.md

```diff
@@ -4,20 +4,22 @@ Create a new ingress for the outpost
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-    name: authentik-outpost
+  name: authentik-outpost
 spec:
-    rules:
-        - host: app.company
-          http:
-              paths: /outpost.goauthentik.io
-              pathType: Prefix
-              backend:
-                  # Or, to use an external Outpost, create an ExternalName service and reference that here.
-                  # See https://kubernetes.io/docs/concepts/services-networking/service/#externalname
-                  service:
-                      name: ak-outpost-example-outpost
-                      port:
-                          number: 9000
+  rules:
+    - host: app.company
+      ingressClassName: nginx
```

Comment:
```suggestion
```

This is very much dependent on the user's kubernetes installation, let's not specify it. Furthermore, it should be `.spec.ingressClassName`, not `.spec.rules.ingressClassName`.